firefox/background.js

```diff
@@ -3,37 +3,56 @@ function generateGoogleLink(tab) {
     const data = url.split("?")[1].split("&")
     const lat = data[0].split("=")[1]
     const lng = data[1].split("=")[1]
+
     const googleLink = `https://www.google.com/maps/@${lat},${lng},18z`
 
     return googleLink
 }
 
+function generateShadowMapLink(tab, isGoogleEarth = false) {
```

Comment:
Very minor suggestion: Use a lower case `M` ;) it's `Shadowmap` not `ShadowMap`.